packages/lib-classifier/src/components/Classifier/components/MetaTools/components/HidePreviousMarksButton/HidePreviousTranscriptionsButton.spec.js

```diff
@@ -0,0 +1,52 @@
+import { shallow } from 'enzyme'
+import React from 'react'
+
+import { Menu } from 'grommet'
+import { HidePreviousTranscriptionsButton, StyledDrop } from './HidePreviousTranscriptionsButton'
+import SHOWN_MARKS from '@helpers/shownMarks'
+
+let wrapper
+
+describe('Component > HidePreviousTranscriptionsButton', function () {
+  before(function () {
+    wrapper = shallow(<HidePreviousTranscriptionsButton />)
+  })
+
+  it('should render without crashing', function () {
+    expect(wrapper).to.be.ok()
+  })
+
+  it('should show the correct text and icon', function () {
```

Comment:
The description here doesn't match what is being tested. Should maybe read `should show the correct aria title`?